src/app/projects/add-edit-project/add-edit-project.component.ts

```diff
@@ -731,6 +732,25 @@ export class AddEditProjectComponent implements OnInit, AfterViewInit, OnDestroy
     });
   }
 
+  /**
+   * Uses the CK Editor (ready) to link a file upload handler to the
+   * CK Editor instance.
+   * 
+   * @param eventData Object type added and used by CK Editor
+   * @returns {Promise}
+   */
+  public editorOnReady(eventData) {
```

Comment:
Great documentation! 

Regarding typescript, I'd like to start explicitly typing function parameters and return types. This would be using Typescript as intended instead of relying on "any". One benefit of typing function params is that less or no defensive coding is needing when taking arguments.

So if we were to type this function, I _think_ the param would be EventInfo(see: https://ckeditor.com/docs/ckeditor5/latest/api/module_core_editor_editor-Editor.html#event-ready). Also I think this function returns `void` and not a Promise. 

```suggestion
  public editorOnReady(eventData: EventInfo): void {
```